custom_components/teslafi/coordinator.py

```diff
@@ -128,3 +138,13 @@ def _infer_charge_session(
                     f"New charge session detected: {prev.charge_session_number} -> {current.charge_session_number}"
                 )
                 self._last_charge_reset = current.last_remote_update
+                
+    async def _update_command_request_counts(
+        self,
+        response: dict
+    ) -> None:
+        """Update request counts."""
+        if response and (request_counts := response.get("tesla_request_counter")):
+            self._vehicle.update_non_empty(request_counts)
```

Comment:
Looks like you're already merging just the counters here, so that's essentially what my pseudocode above was suggesting. So I think all that's missing is to replace the refresh request, with notifying the coordinator that the data has been updated. 

One caveat: that function will reset the polling interval schedule, so if the interval is every 10 minutes, and 1 min before it's scheduled to refresh you send a wake/command and get updated counts, then that update counts as an update of the coordinator and the next polling interval is rescheduled for +10 min. I don't think that's a big deal overall, just something to be aware of.